Tuple4.elm

```diff
@@ -114,20 +116,18 @@ sortWith cmp ( a, b, c, d ) =
     let
         goesBefore x y =
             not <| cmp x y == GT
+
+        ( a_, b_, c_ ) =
+            Tuple3.sortWith cmp ( a, b, c )
```

Comment:
The signature of this function was wrong with the implementation. In the old code on line 130, a tuple with 3 elements is returned.

I found it strange, because this package was published (for `0.17`) after your `Tuple4` addition. I found out why: `Tuple4` wasnt exposed in the `elm-package.json`, and thus not compiled when being packaged. I will add it there.